packages/orbit-components/src/Loading/Loading.stories.tsx

```diff
@@ -1,123 +1,113 @@
 import * as React from "react";
+import type { Meta, StoryObj } from "@storybook/react";
 
 import TYPE_OPTIONS from "./consts";
-import Card from "../Card";
-import CardSection from "../Card/CardSection";
-import Illustration from "../Illustration";
-import Button from "../Button";
 import RenderInRtl from "../utils/rtl/RenderInRtl";
+import Text from "../Text";
 
 import Loading from ".";
 
-export default {
+const meta: Meta<typeof Loading> = {
   title: "Loading",
-};
-
-export const Default = () => <Loading />;
+  component: Loading,
 
-Default.story = {
   parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+    info: "Check Orbit.Kiwi for more detailed design guidelines.",
   },
-};
-
-export const ButtonLoading = () => <Button loading>Default button</Button>;
 
-ButtonLoading.story = {
-  name: "Button loading",
-
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  args: {
+    loading: true,
+    text: <Text>Please wait, content is loading...</Text>,
+    customSize: 50,
+    type: TYPE_OPTIONS.PAGE_LOADER,
+    id: "loader-id",
   },
-};
 
-export const CardLoading = ({ title, description, loading, loadingText }) => {
-  return (
-    <Card loading={loading} title={title} description={description}>
-      <Loading loading={loading} type="boxLoader" text={loadingText}>
-        <CardSection>
-          <Illustration name="EnjoyApp" size="medium" />
-        </CardSection>
-      </Loading>
-    </Card>
-  );
-};
-
-CardLoading.story = {
-  name: "Card loading",
-
-  parameters: {
-    info: "This is the default configuration of this component. Visit Orbit.Kiwi for more detailed guidelines.",
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
   },
 };
 
-CardLoading.args = {
-  title: "Card with title",
-  description: "",
-  loading: true,
-  loadingText: "Please wait, Card content is loading...",
-};
+export default meta;
 
-export const InlineLoader = ({ loadingText }) => {
-  return <Loading type={TYPE_OPTIONS.INLINE_LOADER} text={loadingText} />;
-};
+type Story = StoryObj<typeof Loading>;
 
-InlineLoader.story = {
-  name: "Inline loader",
+export const Default: Story = {
+  args: {
+    customSize: undefined,
+    loading: undefined,
+    text: undefined,
+    type: undefined,
+  },
 
   parameters: {
-    info: "This configuration of this component is without any height, width or paddings. Visit Orbit.Kiwi for more detailed guidelines.",
+    controls: {
+      disable: true,
+    },
   },
 };
 
-InlineLoader.args = {
-  loadingText: "Please wait, content of the page is loading...",
-};
+export const DefaultWithChildren: Story = {
```

Comment:
What do you suggest to add there instead of the basic text component? I just added this `<Text>` component for simple visualization, but I can add there more (eg. illustration, alert components, whatever,...)